rocks/openstack-exporter/rockcraft.yaml

```diff
@@ -3,7 +3,7 @@ summary: Openstack openstack-exporter
 license: Apache-2.0
 description: |
   Ubuntu distribution of openstack-exporter, a prometheus exporter for OpenStack
-version: 1.7.0
+version: 1.7.0-3be9ddb
```

Comment:
This is a non-blocker comment.

nit: I suggest changing to `1.7.0-26-g3be9ddb` because this align with the output of:

```shell
$ git describe --tags
v1.7.0-26-g3be9ddb
```

Reply:
Currently not naming after a tag, but a version.

As you can see, the true tag was `v1.7.0`

Reply:
Since the tag was added several commits were merged and no new release was done. IMHO, this can mislead a little bit to users think that it's the actual [v1.7.0](https://github.com/openstack-exporter/openstack-exporter/releases/tag/v1.7.0) release, which is not.

Fell free to resolve this conversation.

Reply:
The primary goal is to inform from where this version comes from. it's 1.7.0 + something else. It's that we have done in the past, and I'm not keen on changing this.